bin/setup

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+brew install carthage 2> /dev/null
```

Comment:
Could grab brew_install_or_upgrade and its helper functions from https://github.com/thoughtbot/laptop/blob/master/mac